acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -115,6 +123,10 @@ private TekuNode(final Network network, final DockerVersion version, final Confi
         .withCommand("--config-file", CONFIG_FILE_PATH);
   }
 
+  public Spec getSpec() {
+    return spec;
+  }
```

Comment:
This doesn't feel like something that should be public... I think it's not needed if the BLS change creator gets pushed into the DSL.